### DISCUSSION TITLE
Standard link type on homescreen

### DISCUSSION THREAD

**Discussion by bonelifer** [jcorporation/myMPD#785] [Ideas]
Add a standard link type to homescreen, so we can add various local/non-local music related links to the homepage. One example is I have mpd3chromecast installed and it would be nice to link to http://localhost:8070 so that I can enable/disable casting MPD HTTP streams to my Chromecast with Google TV dongle as needed. Another example, is I use BlissHQ to manage my music collection data. These are just two examples.

**Comment by jcorporation** [comment]
This should be relatively easy to implement. At the moment there is no central add button for the homescreen. Where should be the action button to add these type of home icon?

**Comment by bonelifer** [comment]
I would say on the homescreen itself.  No where else seems right.

**Reply by jcorporation** [reply]
This implies a toolbar for the homescreen with only one button

**Comment by bonelifer** [comment]
You're right, better to add it to the side menu, above add streams. Just getting used to the UI. So "add link" or "add link to homescreen".

**Comment by jcorporation** [comment]
This action is for a home menu entry too seldom used. Another option is a contextmenu on the homescreen… or there are other useful buttons for a toolbar.

**Comment by jcorporation** [comment]
Feature is now implemented in the v9.4.0 branch. There is a new small settings button in the top right corner. That opens a dropdown menu with the option to add an external link.
